### DISCUSSION TITLE
recorded flows to pcap

### DISCUSSION THREAD

**Discussion by BMBurstein** [mitmproxy/mitmproxy#7042] [Q&A] [answered]
I have a saved flows file that I would like to convert to a pcap file. All my searching seems to find how to record a live mitm session with wireshark using SSLKEYLOGFILE. How do I cinvert an existing flows file that was recorded without SSLKEYLOGFILE?

**Comment by mhils** [comment, marked as answer]
Converting mitmproxy flow files to PCAP is not possible. If you need a PCAP, you need to record it with Wireshark/tcpdump.

The one thing you _can_ do with mitmproxy is ask it to produce an SSLKEYLOGFILE: https://docs.mitmproxy.org/stable/howto-wireshark-tls/
